content/pytorch/concepts/tensors/terms/linspace/linspace.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: '.linspace()'
+Description: 'Returns a one-dimensional tensor of steps equally spaced points between start and end.'
+Subjects:
+  - 'Data Science'
+  - 'Machine Learning'
+Tags:
+  - 'AI'
+  - 'Data Types'
+  - 'Deep Learning'
+  - 'Functions'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'py-torch-for-classification'
+---
+
+The **`.linspace()`** function in Pytorch is used to create a one-dimensional tensor of steps equally spaced points between the start and end values. It takes three arguments: the start value, the end value, and the number of steps.
+
+The function returns a one-dimensional tensor of equally spaced points between the start and end values.

```

Comment:
Modifications:

- Let's remove the second line as it's already mentioned in `Syntax`
- Let's remove the third line as it resembles the first line
- Let's change the first line a bit for uniqueness

```suggestion
The **`.linspace()`** function in PyTorch is used to return a one-dimensional tensor of the specified size, whose values are equally spaced from the given starting and ending values.
```